Activities/Database/ConnectionDialog/ConnectionUIDialog/Workaround/DbWorkarounds.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Microsoft.Data.ConnectionUI;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace UiPath.Database.Workaround
+{
+    public static class DbWorkarounds
+    {
+        [DllImport("kernel32.dll", CharSet = CharSet.Auto, SetLastError = true)]
+        private static extern IntPtr LoadLibrary(string libname);
+
+        [DllImport("kernel32.dll", CharSet = CharSet.Auto)]
+        private static extern bool FreeLibrary(IntPtr hModule);
+
+
+        public static void SNILoadWorkaround()
+        {
+#if NETCOREAPP
```

Comment:
These can be unified, just the wix-x64/x86 is different between them.

Same for the other instance of this class.

Reply:
unified the lines, only the load library is different now depending on .net version